cro-clib/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2018"
 
 [lib]
 name = "cro_clib"
-crate-type =["staticlib"]
+crate-type =["staticlib", "cdylib"]
```

Comment:
dynamic library is intentionally removed.
because of ssl related error

Reply:
@yihuang is this necessary? if it is, these instructions will need update: https://github.com/crypto-com/chain/blob/master/cro-clib/android.md as one may need to remove `cdylib` manually

Reply:
How about moving the c wrapper for jsonrpc API into the `client-rpc-core` crate, produce `cdylib` there. `cro-clib` produce staticlib and cross-compiling as before.

Reply:
I think that works -- probably https://github.com/crypto-com/chain/blob/master/cro-clib/build.rs will need updating?

Reply:
Removed the `cdylib`, change the `crate-type` in `build.sh` with tool on the fly to build the shared library.